docs/getting_started_guide/ISO_Build/Environment_Preparation.rst

```diff
@@ -6,52 +6,24 @@ Environment Preparation
 Getting Started
 ---------------
 
-.. NOTE::
-   You can skip setting up ``mock`` if you are generating an ISO from a release tarball!
+.. WARNING::
 
-Required Packages
-^^^^^^^^^^^^^^^^^
+   Please use a **non-root** user for building SIMP!
 
-Before we proceed, you'll want to make sure that you have a few :term:`RPM`
-packages installed on your system and that your build system has access to the
-Internet. If you're using :term:`Enterprise Linux` as your build system, you
-will need to enable :term:`EPEL` for your system prior to proceeding.
+Ensure Sufficient Entropy
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Required RPMs
-"""""""""""""
+The SIMP build generates various keys and does quite a bit of package
+signing. As such, your system must be able to keep its entropy pool
+full at all times. If you check ``/proc/sys/kernel/random/entropy_avail``
+and it shows a number below **1024**, then you should either make sure that
+``rngd`` is running and pointed to a hardware source (preferred) or install
+and use **haveged**.
 
 .. code-block:: bash
 
-  # Installing from EL 6:
-
-    $ sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
-    $ sudo yum install -y augeas-devel createrepo genisoimage git gnupg2 \
-        libicu-devel libxml2 libxml2-devel libxslt libxslt-devel \
-        mock rpm-sign rpmdevtools clamav gcc gcc-c++ ruby-devel
-
-  # Installing from EL 7:
-
-    $ sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-    $ sudo yum install -y augeas-devel createrepo genisoimage git gnupg2 \
-        libicu-devel libxml2 libxml2-devel libxslt libxslt-devel \
-        mock rpm-sign rpmdevtools clamav-update gcc gcc-c++ ruby-devel
-
-  # Installing from Fedora 23/24:
-
-    $ sudo dnf install -y augeas-devel createrepo genisoimage git gnupg \
-        libicu-devel libxml2 libxml2-devel libxslt libxslt-devel \
-        mock rpm-sign rpmdevtools clamav-update gcc gcc-c++ ruby-devel
-
-.. WARNING::
-   Please use a **non-root** user for building SIMP!
-
-.. NOTE::
-   The SIMP build generates various keys and does quite a bit of package
-   signing. As such, your system must be able to keep its entropy pool
-   full at all times. If you check ``/proc/sys/kernel/random/entropy_avail``
-   and it shows a number below **1024**, then you should either make sure that
-   ``rngd`` is running and pointed to a hardware source (preferred) or install
-   and use **haveged**.
+  $ yum install haveged
+  $ systemctl start haveged
```

Comment:
* sudo yum install .... sudo systemctl ...
* Do you want to have them enable haveged as well?
   sudo systemctl enable haveged